app/assets/javascripts/models/data/collection.js.coffee

```diff
@@ -62,6 +62,7 @@ ns.Collection = do (ko
       @details = @asyncComputed({}, 100, @_computeCollectionDetails, this)
       @detailsLoaded = ko.observable(false)
       @gibs = ko.observable(null)
+      @gibsLayers = ko.observable(null)
```

Comment:
The problem with this is that the available layers are given in the fromJson function.  Since it is dependent upon the JSON data that function receives, I'm not sure how to break it out of that function.